release/default.nix

```diff
@@ -2,19 +2,19 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-{ pkgs ? import (import ../nix/sources.nix).nixpkgs { } }:
+{ pkgs ? import (import ../nix/sources.nix).nixpkgs { }, timestamp }:
 let
   executable = (import ../xrefcheck.nix {
     static = true;
   }).components.exes.xrefcheck;
-  joinByBasename = name: paths:
-    pkgs.runCommandNoCC name { } (''
-      mkdir $out
-    '' + pkgs.lib.concatMapStrings (path: ''
-      ln -s ${path} $out/${builtins.baseNameOf path}
-    '') paths);
-in joinByBasename "xrefcheck-release" [
-  "${executable}/bin"
-  ../LICENSES
-  ../README.md
-]
+
+  releaseNotes = pkgs.writeTextDir "release-notes.md" ''
+    Automatic release on ${timestamp}
+  '';
+in pkgs.buildEnv {
+  name = "xrefcheck-release";
+  paths = [
+    releaseNotes
+    "${executable}/bin"
+  ];
+}
```

Comment:
Also I would suggest renaming the executable in release artifacts from `xrefcheck` to `xrefcheck-linux-x86_64-static-binary`. I can imagine that one day we may have other artifacts and `xrefcheck` may become ambiguous, so we will have to rename it to something longer and it will break `wget -O xrefcheck bla-bla && ./xrefcheck`. Moreover, this name is simply more descriptive, it makes it clear that this thing is a static binary for linux x86_64.

Reply:
I think long executable names (especially with `static-binary` in the end) aren't a very good idea. Would you be satisfied with `xrefcheck-x86_64-linux` ?

Reply:
Yes, I am fine with `xrefcheck-x86_64-linux`. It'd be nice if github releases had something like symlinks so that we could have both `xrefcheck-x86_64-linux` and `xrefcheck` without duplicating this file, but I guess there is no such feature :)